src/app/ai/page.jsx

```diff
@@ -84,11 +84,11 @@ const AIPage = () => (
       className="mt-[180px] 2xl:mt-40 xl:mt-[125px] lg:mt-16"
       buttonClassName="px-14 xl:px-10 lg:px-9 sm:px-14"
       title="Build your next AI app now with Neon"
-      description="Neon offers flexible usage and volume-based plans. Contact our Sales team to learn more."
-      buttonText="Contact Sales"
-      buttonUrl={LINKS.contactSales}
-      linkText="Learn more on GitHub"
-      linkUrl="https://github.com/neondatabase/pg_embedding"
+      description="Build your AI app on our Free Tier. Upgrade to a paid plan when you're ready to scale."
+      buttonText="Start for Free"
+      buttonUrl={LINKS.signup}
+      linkText="View the docs"
+      linkUrl="https://neon.tech/docs/ai/ai-intro"
```

Comment:
this somehow on the deployment @saimonkat shared redirects to console!

Reply:
@rishi-raj-jain so, it's not redirect, we just have this link in the contacts, isn't it?
`signup: 'https://console.neon.tech/signup',`

Reply:
oh i got confused with linkUrl and buttonUrl

Reply:
@danieltprice is this expected flow?

Reply:
Yeah, I just changed the button link from the Contact Sales page to the Neon sign-up page. I changed the URL link to the AI docs instead of the sunsetted pg_embedding repo.
Looks good to me!